docs/coding-guidelines/coding-style.md

```diff
@@ -24,6 +24,7 @@ The general rule we follow is "use Visual Studio defaults".
 9. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
    rather than `_member`), the existing style in that file takes precedence.
 10. We only use `var` when the type is explicitly named on the right-hand side, typically due to either `new` or an explicit cast, e.g. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`.
+    - Similarly, target-typed `new()` can only be used when the type is explicitly named on the left-hand side, in a variable definition statement or a field definition statement. e.g. `FileStream stream = new(...);`, but not `stream = new(...);` (where the type was specified on a previous line).
```

Comment:
Is there an `.editorconfig` option for this? If not, can we open a tracking issue against Roslyn to ensure one is created?

Reply:
@CyrusNajmabadi - is that what https://github.com/dotnet/roslyn/issues/55305 is tracking? Or is there already an .editorconfig for this coding style, and I just can't find it?

Reply:
we already have an editorconfig option to support what the C# team viewed as 'apparent' here.  It is called csharp_style_implicit_object_creation_when_type_is_apparent.  I think you should use that and our rules instead of a different set of rules.

Reply:
> I think you should use that and our rules instead of a different set of rules.

The rule we defined here in the dotnet/runtime is consistent with the rule of when we allow `var`. Why should we have inconsistent rules in our codebase?